README.md

```diff
@@ -45,6 +45,103 @@ Required installed programs: Bash and jq for bash, Python3 and the PyVCF, filelo
+- **Total SNPs** -- The total number of SNPs used in the calculation
+- **Used Super Population** -- The super population used for linkage disequillibrium
+
+#### Columns Only Available In The Full Version
+- **Percentile** -- Indicates the percentile rank of the samples polygenic risk score
+- **Protective Variants** -- Variants that are protective against the phenotype of interest
+- **Risk Variants** -- Variants that add risk for the phenotype of interest
+- **Variants Without Risk Alleles** -- Variants that are present in the study, but the sample does not possess the allele reported with association
+- **Variants in High LD** -- Variants that are not used in the calculation, due to them being in high linkage disequillibrium with another variant in the study.
+
+### Condensed
+
+This version of the output results contains one row for each study with columns for each sample's polygenic risk score. A column will be named using the samples identifier and that column will hold their risk scores. 
+
+Study ID | Reported Trait | Trait | Citation | P-Value Annotation | Beta Annotation | Score Type | Units (if applicable) | SNP Overlap | SNPs Excluded Due To Cutoffs | Total SNPs | Used Super Population | Sample1 | Sample2 | Sample3 | ect. 
+
+.. code-block:: bash
+
+   ./runPrsCLI.sh -f path/to/file/samples.vcf -o path/to/file/output.tsv -c 0.0005 -r hg19 -p SAS
+
+### Full
+
+This version of the output results contains one row for each sample/study pair. It also includes columns listing the rsIDs of the snps involved in the risk score calculation. 
+
+Sample | Study ID | Reported Trait | Trait | Citation | P-Value Annotation | Beta Annotation | Score Type | Units (if applicable) | SNP Overlap | SNPs Excluded Due To Cutoffs | Total SNPs | Used Super Population | Polygenic Risk Score | Protective Variants | Risk Variants | Variants Without Risk Allele | Variants in High LD
+
+.. code-block:: bash
+
+   ./runPrsCLI.sh -f path/to/file/samples.vcf -o path/to/file/output.tsv -c 0.0005 -r hg19 -p SAS -v
+
+### JSON
+
+This version outputs the results in a json object format. The output automatically contains all the data the full version does and there is no condensed version of the json output. The file will contain a list of json study objects. Each study object will contain the list of samples and their score. 
+
+.. code-block:: bash
+    # example output
+    [
+        {
+            "studyID": "GCST001",
+            "reportedTrait": "Alzheimer's Disease",
+            "trait": "Alzheimer Disease",
+            "citation": "First Author et al. 2021",
+            "pValueAnnotation": "NA",
+            "betaAnnotation": "NA",
+            "scoreType": "OR",
+            "units (if applicable)": "NA",
+            "snpOverlap": 8,
+            "excludedSnps": 3,
+            "totalSnps": 14,
+            "usedSuperPop": "EUR",
+            "samples": [
```

Comment:
A few questions on how the riskAlleles, variantsWithoutRiskAllele, and variantsInHighLD columns work, relating to this example:
1. With SAMP002 as an example (and the other two as well I guess), if 3 SNPs are excluded from the study, will those SNPs still show up in the variantsWithoutRiskAllele column (or in any of the other columns for that matter)? My expectation is that they would not show up in any of the columns (or in an excluded SNPs column).
2. Will the variantsInHighLD column be the same across all samples, or will it be like SAMP002 and SAMP003 where they are different? My expectation is that they would all be the same.
3. In SAMP003, I expect that the riskAlleles column will not have any overlapping SNPs with the variantsWithoutRiskAllele column. Is that a correct assumption to make?

Reply:
1. that is correct
2. using sample-wide clumping, yes they would be the same. I think if you use individual-clumping it could potentially be different.
3. Not necessarily. If someone is heterozygous for the risk allele, then they could have 1 snp in riskAlleles and 1 snp in variantsWithoutRiskAllele. Is it confusing to have it that way? I thought it would be helpful to do it like that to indicate why 2 individuals with the same snps but one is homozygous and one is heterozygous have different scores.

Reply:
1. 3 of the 14 snps need to not show up in each of the samples in this example
2. SAMP002 needs rs12 and rs13 moved to its variantsInHighLD section.
3. Oh! I see that now. That is not initially intuitive, but makes sense after you have explained it. I think you should keep it this way, but explain above why this overlap occurs sometimes.

Reply:
Okay I added an explanation and made those changes